### PR TITLE
README: Add `author_email`

### PR DESCRIPTION
**Pull request by Asnelchristian**
This will certainly fix the issue with pypi.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -78,6 +78,7 @@ def run(self):
           version=VERSION,
           description='Code Analysis Application (coala)',
           author="The coala developers",
+          author_email="coala.io",
```
Comment (sils1297) [inline setup.py]: that's not really a email address

Comment (Asnelchristian) [inline setup.py]: i was awaiting that comment :P

Comment (sils1297) [inline setup.py]: use coala.analyzer@gmail.com

File: setup.py [hunk 2]
```diff
@@ -78,8 +78,9 @@ def run(self):
           version=VERSION,
           description='Code Analysis Application (coala)',
           author="The coala developers",
+          author_email="coala.analyzer@gmail.com",
           maintainer="Lasse Schuirmann, Fabian Neuschmidt, Mischa Kr\xfcger"
-                      if not on_rtd else "L.S., F.N., M.K.",
+          if not on_rtd else "L.S., F.N., M.K.",
```
Comment (sils1297) [inline setup.py]: why do you change this line?

Comment (Asnelchristian) [inline setup.py]: ooh i didn't, maybe that is PEP8Bear

**Comment by sils1297** [comment]
ack 6c6309a

**Comment by Asnelchristian** [comment]
@rultor merge

**Pull request merged**